src/applications/check-in/components/tests/TravelPayOMB.unit.spec.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { expect } from 'chai';
+import { render } from '@testing-library/react';
+
+import TravelPayOMB from '../TravelPayOMB';
+
+describe('check-in', () => {
+  describe('TravelPayOMB', () => {
+    it('renders the OMB information correctly', () => {
+      const { getByTestId, getByText } = render(<TravelPayOMB />);
+
+      // Verify the `va-omb-info` element exists with the correct data-testid
+      const ombInfo = getByTestId('travel-pay-omb');
+      expect(ombInfo).to.exist;
+
+      // Verify the `exp-date` prop
+      expect(ombInfo.getAttribute('exp-date')).to.equal('11/30/2027');
+
+      // Verify the `omb-number` prop
+      expect(ombInfo.getAttribute('omb-number')).to.equal('2900-0798');
+
+      // Verify the `res-burden` prop
+      expect(ombInfo.getAttribute('res-burden')).to.equal('10');
+
+      // Verify text content within the component
+      expect(getByText(/VA Burden Statement:/)).to.exist;
+      expect(
+        getByText(
```

Comment:
In the check-in app we made the decision a while back to favor test ids over exact strings in our tests(could be up for debate). You could add test ids to the paragraph tags and since the text itself is using props passed in you could test for just those strings like: 
```
expect(getByText(/2900-0798/)).to.exist;
```

Reply:
Ahh good to know. I'll update my tests accordingly